src/micromamba/install.cpp

```diff
@@ -285,8 +273,8 @@ install_specs(const std::vector<std::string>& specs, bool create_env, int solver
     bool success = solver.solve();
     if (!success)
     {
-        std::cout << "\n" << solver.problems_to_str() << std::endl;
```

Comment:
can we revert these changes here?

This is not a log message, it's an output

Reply:
Although in the future we might want to adjust this so that we get the proper json output

Reply:
But LOG_ERROR has implications also with regard to `--json` and `--quiet`

Reply:
done